drools-game-engine-services/src/test/java/org/drools/game/services/endpoint/test/CaptureTheFlagTest.java

```diff
@@ -0,0 +1,115 @@
+    private ResteasyWebTarget target;
+    private static final String APP_URL = "http://localhost:8080/";
+    
+    @BeforeClass
+    public static void setupCmds() {
+    	// FIXME Relocated temporarily on StoreFactsTestImp
+    	
+//        CommandRegistry.set( "TELEPORT_CALLBACK", "org.drools.game.services.endpoint.test.cmds.TeleportPlayerCommand" );
+//        CommandRegistry.set( "CLEAR_INVENTORY_CALLBACK", "org.drools.game.services.endpoint.test.cmds.ClearPlayerInventoryCommand" );
+//        CommandRegistry.set( "NOTIFY_VIA_CHAT_CALLBACK", "org.drools.game.services.endpoint.test.cmds.NotifyViaChatCommand" );
+//        CommandRegistry.set( "NOTIFY_ALL_VIA_CHAT_CALLBACK", "org.drools.game.services.endpoint.test.cmds.NotifyAllViaChatCommand" );
+//        CommandRegistry.set( "RESET_FLAG_CALLBACK", "org.drools.game.services.endpoint.test.cmds.ResetFlagCommand" );
+//        CommandRegistry.set( "SET_PLAYER_HEALTH_CALLBACK", "org.drools.game.services.endpoint.test.cmds.SetPlayerHealthCommand" );
+//        CommandRegistry.set( "SET_PLAYER_PARAM_CALLBACK", "org.drools.game.services.endpoint.test.cmds.SetPlayerParamCommand" );
+    }
+    
+    @Before
+    public void setup() throws Exception {
+        client = ClientBuilder.newBuilder().build();
+        final WebTarget webtarget = client.target( APP_URL );
+        target = ( ResteasyWebTarget ) webtarget;
+        
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        client.close();
+    }
+    
+    @Deployment(testable=false)     
+    public static Archive createDeployment() throws IllegalArgumentException, Exception {
+        JAXRSArchive deployment = ShrinkWrap.create( JAXRSArchive.class );
+        deployment.addPackages( true, "com.google.common" );
+        deployment.addPackage( "org.drools.game.services.endpoint.test.cmds" );
+        deployment.addClass( GameService.class );
+        deployment.addClass( GameServiceImpl.class );
+        deployment.addClass( GameSessionInfo.class );
+        deployment.addClass( StoreFacts.class );
+        deployment.addClass( StoreFactsTestImp.class );
+        deployment.addClass( CommandRest.class );
+        deployment.addClass( Player.class );
+        
+        deployment.addAllDependencies()
+                .addAsManifestResource( EmptyAsset.INSTANCE, "beans.xml" );
+        
+        return deployment;
+    }
+    
+	@Test
+	public void pickTheFlagHitTheChasmTest() {
+		
+		GameService game = target.proxy(GameService.class);
+
+		String sessionId = game.newGameSession();
+		String playerName = "salaboy";
+        Player player = new BasePlayerImpl( playerName );
+
+        game.joinGameSession(sessionId, player );
+
+        Flag flag = new Flag( "Flag", "banner" );
+        Zone chasm = new Zone( "Chasm" );
+  
+        game.execute(sessionId, new CommandRest( new PickFlagCommand( player, flag )));
+        game.execute(sessionId, new CommandRest( new EnterZoneCommand( player, chasm )));
```

Comment:
This is perfect, we just need to remove the CommandRest that I believe that is not necessary.